engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -20,9 +20,10 @@
 import java.net.URI;
 import java.nio.ByteBuffer;
 import java.util.*;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.ConcurrentLinkedQueue;
 import java.util.concurrent.ConcurrentMap;
-import java.util.concurrent.atomic.AtomicInteger;
+import java.util.Map.Entry;
```

Comment:
You are right :) But if I move it before java.util.concurrent the checkstyle errors are increased..